staging/src/k8s.io/client-go/rest/fake/fake.go

```diff
@@ -87,23 +89,27 @@ func (c *RESTClient) GetRateLimiter() flowcontrol.RateLimiter {
 }
 
 func (c *RESTClient) request(verb string) *restclient.Request {
-	config := restclient.ContentConfig{
-		ContentType: runtime.ContentTypeJSON,
-		// TODO this was hardcoded before, but it doesn't look right
-		GroupVersion:         &c.APIRegistry.GroupOrDie("").GroupVersion,
-		NegotiatedSerializer: c.NegotiatedSerializer,
+	config := c.ContentConfig
+	defaultContentConfig := restclient.ContentConfig{}
```

Comment:
See the unit test that applies deployment objects (also in the extensions group)